modules/shapefile/test/shp-loader.spec.js

```diff
@@ -1,17 +1,15 @@
 import test from 'tape-promise/tape';
-import {load, fetchFile} from '@loaders.gl/core';
+import {load, loadInBatches, fetchFile} from '@loaders.gl/core';
 import {geojsonToBinary} from '@loaders.gl/gis';
 import {SHPLoader} from '@loaders.gl/shapefile';
 
-// import {parseSHP} from '@loaders.gl/shapefile/lib/parsers/parse-shp';
-// // import parseSHP from '@loaders.gl/shapefile/lib/parsers/deprecated/parse-shp-atomic';
-// import {fetchFile} from '@loaders.gl/core';
-
 const SHAPEFILE_POLYGON_PATH = '@loaders.gl/shapefile/test/data/shapefile-js/polygons.shp';
 const SHAPEFILE_JS_DATA_FOLDER = '@loaders.gl/shapefile/test/data/shapefile-js';
 const SHAPEFILE_JS_POINT_TEST_FILES = ['points', 'multipoints'];
 const SHAPEFILE_JS_POLYLINE_TEST_FILES = ['polylines'];
 const SHAPEFILE_JS_POLYGON_TEST_FILES = ['polygons'];
+const SHAPEFILE_CONTOURS_PATH =
+  '@loaders.gl/shapefile/test/data/other/geo_export_14556060-0002-4a9e-8ef0-03da3e246166.shp';
```

Comment:
Passes locally on this file. Should I take the test out? Some bugs like those that prompted this PR can't be found with such small files that are currently used for tests, because those have only one batch

Reply:
You could load this file from rawgithubuserdata link, served from the github repo. it is not great to have tests depend on network loads.

But another reason for not including this in standard runs is also that tests should be fast.

Reply:
> But another reason for not including this in standard runs is also that tests should be fast.

That makes a lot of sense... So should I just take this out? Is there a way to only have a test run on CI and not locally? That might make things confusing though.

Reply:
You could probably have a special entry point for tests that includes a few extra test cases `test-live` or something.

You could add this to the benchmarks, and do some assertions there after loading.